version.props

```diff
@@ -1,7 +1,9 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>2.1.0</VersionPrefix>
-    <ExperimentalVersionPrefix>0.4.0</ExperimentalVersionPrefix><ExperimentalPackageVersion Condition="'$(IsFinalBuild)' == 'true'">$(ExperimentalVersionPrefix)-$(ExperimentalVersionSuffix)-final</ExperimentalPackageVersion>
+    <ExperimentalVersionPrefix>0.4.0</ExperimentalVersionPrefix>
+    <ExperimentalVersionSuffix>alpha1</ExperimentalVersionSuffix>
+    <ExperimentalPackageVersion Condition="'$(IsFinalBuild)' == 'true'">$(ExperimentalVersionPrefix)-$(ExperimentalVersionSuffix)-final</ExperimentalPackageVersion>
```

Comment:
We don't need this now, but for RTM we'll need to check if `'$(ExperimentalVersionSuffix)' == 'rtm' if we want to produce just `0.4.0` as the package version.